### PR TITLE
PSE #1

### PR DESCRIPTION
**Pull request by CheezItMan**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -8,32 +8,24 @@
...
-      -
-        Type: Lesson
-        UID: ece6a3da4a290866251b1bd78d1dbb37
-        Path: /02-arrays/01-arrays.md
       -
         Type: Checkpoint
-        UID: b3e5030f89373dae804212918d12cf94
-        Path: /02-arrays/02-arrays-checkpoint.md
+        UID: 1caa645b-5306-4ba3-b37f-c02d11bb680e
+        Path: /01-pse-01/01-wk-02-pse-checkpoint.md
```
Comment (tildeee) [inline config.yaml]: Nitpick: I think the naming convention is `01-wk02-pse-checkpoint.md` (no dash between `wk` and `02`)

File: config.yaml [hunk 2]
```diff
@@ -8,32 +8,24 @@
...
-        Type: Checkpoint
-        UID: 45f708d7a65ca33c0e8e298ae804f9bd
-        Path: /01-binary-numbers/02-binary-numbers-checkpoint.md
+        Path: 00-about-pses/01-about-pses.md
   -
-    Title: Arrays
-    Description: Arrays
-    UID: ff43b8de4f41d5103405ddb62eb8d34e
+    Title: PSE 1
+    Description: PSE 01
```
Comment (tildeee) [inline config.yaml]: Can we have a blank description? Otherwise it'll just have unnecessary text on Learn

**Review by tildeee — CHANGES_REQUESTED** [review]
I think you forgot to add the actual files 😅  Re-request a review when you're ready!

File: config.yaml (hunk 2 above)
Comment (kaidamasaki) [reply config.yaml]:
```suggestion
    Description: Problem Solving Exercise 1
```

File: config.yaml (hunk 1 above)
Comment (kaidamasaki) [reply config.yaml]: Nitpick: use a suggestion for your nitpicks.  :-P

```suggestion
        Path: /01-pse-01/01-wk02-pse-checkpoint.md
```

Comment (tildeee) [reply config.yaml]: It'd probably be a more cohesive commit to change the config path and the file name in the same commit

**Pull request merged**